ResidentsSocialCarePlatformApi.Tests/V1/Helper/TestHelper.cs

```diff
@@ -86,8 +83,8 @@ public static NoteType CreateDatabaseNoteType(string noteType = "CASSUMASC", str
             var faker = new Fixture();
 
             return faker.Build<NoteType>()
-                .With(noteType => noteType.Type, noteType)
-                .With(noteType => noteType.Description, description)
+                .With(noteTypeI => noteTypeI.Type, noteType)
+                .With(noteTypeI => noteTypeI.Description, description)
```

Comment:
What was the reason we made this change?

Reply:
because the variable noteType was also a parameter it was hiding the outer variable. Admittedly it didn't matter but I think as best practice it's best to avoid essentially overriding the variable name.

Reply:
Ahhhhh sorry, I didn't spot that - makes sense!